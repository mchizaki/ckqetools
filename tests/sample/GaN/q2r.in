&input
  fildyn = 'GaN.dyn'
  flfrc  = 'GaN.fc'
/

