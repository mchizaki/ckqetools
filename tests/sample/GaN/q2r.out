
     Program Q2R v.7.2 starts on 10Jul2024 at 12:47:10 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      96 processor cores
     Number of MPI processes:                96
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      96
     88317 MiB available memory on the printing compute node when the environment starts
 

  reading grid info from file GaN.dyn0    

  reading force constants from file GaN.dyn1
 macroscopic fields = T
   5.34262   0.00000   0.00000
   0.00000   5.34262   0.00000
   0.00000   0.00000   5.80678
  na=            1
   2.62562   0.00000   0.00000
   0.00000   2.62562   0.00000
   0.00000   0.00000   2.76550
  na=            2
   2.62562   0.00000   0.00000
   0.00000   2.62562   0.00000
   0.00000   0.00000   2.76550
  na=            3
  -2.51614   0.00000   0.00000
   0.00000  -2.51614   0.00000
   0.00000   0.00000  -2.76521
  na=            4
  -2.51614   0.00000   0.00000
   0.00000  -2.51614   0.00000
   0.00000   0.00000  -2.76521
  nqs=            1
 q=   0.00000000  0.00000000  0.00000000
  reading force constants from file GaN.dyn2
  nqs=            2
 q=   0.00000000  0.00000000  0.15353963
 q=   0.00000000  0.00000000 -0.15353963
  reading force constants from file GaN.dyn3
  nqs=            1
 q=   0.00000000  0.00000000 -0.30707926
  reading force constants from file GaN.dyn4
  nqs=            6
 q=   0.00000000  0.28867514  0.00000000
 q=   0.00000000 -0.28867514  0.00000000
 q=  -0.25000000 -0.14433757  0.00000000
 q=   0.25000000 -0.14433757  0.00000000
 q=  -0.25000000  0.14433757  0.00000000
 q=   0.25000000  0.14433757  0.00000000
  reading force constants from file GaN.dyn5
  nqs=           12
 q=   0.00000000  0.28867514  0.15353963
 q=   0.00000000 -0.28867514 -0.15353963
 q=   0.00000000 -0.28867514  0.15353963
 q=   0.00000000  0.28867514 -0.15353963
 q=  -0.25000000 -0.14433757  0.15353963
 q=   0.25000000  0.14433757 -0.15353963
 q=   0.25000000 -0.14433757  0.15353963
 q=  -0.25000000  0.14433757 -0.15353963
 q=  -0.25000000  0.14433757  0.15353963
 q=   0.25000000 -0.14433757 -0.15353963
 q=   0.25000000  0.14433757  0.15353963
 q=  -0.25000000 -0.14433757 -0.15353963
  reading force constants from file GaN.dyn6
  nqs=            6
 q=   0.00000000  0.28867514 -0.30707926
 q=   0.00000000 -0.28867514 -0.30707926
 q=  -0.25000000 -0.14433757 -0.30707926
 q=   0.25000000 -0.14433757 -0.30707926
 q=  -0.25000000  0.14433757 -0.30707926
 q=   0.25000000  0.14433757 -0.30707926
  reading force constants from file GaN.dyn7
  nqs=            3
 q=   0.00000000 -0.57735027  0.00000000
 q=   0.50000000  0.28867514  0.00000000
 q=  -0.50000000  0.28867514  0.00000000
  reading force constants from file GaN.dyn8
  nqs=            6
 q=   0.00000000 -0.57735027  0.15353963
 q=   0.00000000  0.57735027 -0.15353963
 q=   0.50000000  0.28867514  0.15353963
 q=  -0.50000000 -0.28867514 -0.15353963
 q=  -0.50000000  0.28867514  0.15353963
 q=   0.50000000 -0.28867514 -0.15353963
  reading force constants from file GaN.dyn9
  nqs=            3
 q=   0.00000000 -0.57735027 -0.30707926
 q=   0.50000000  0.28867514 -0.30707926
 q=  -0.50000000  0.28867514 -0.30707926
  reading force constants from file GaN.dyn10
  nqs=            6
 q=   0.25000000  0.43301270  0.00000000
 q=   0.25000000 -0.43301270  0.00000000
 q=  -0.25000000 -0.43301270  0.00000000
 q=  -0.50000000  0.00000000  0.00000000
 q=  -0.25000000  0.43301270  0.00000000
 q=   0.50000000  0.00000000  0.00000000
  reading force constants from file GaN.dyn11
  nqs=           12
 q=   0.25000000  0.43301270  0.15353963
 q=  -0.25000000 -0.43301270 -0.15353963
 q=   0.25000000 -0.43301270  0.15353963
 q=  -0.25000000  0.43301270 -0.15353963
 q=  -0.25000000 -0.43301270  0.15353963
 q=   0.25000000  0.43301270 -0.15353963
 q=  -0.50000000  0.00000000  0.15353963
 q=   0.50000000  0.00000000 -0.15353963
 q=  -0.25000000  0.43301270  0.15353963
 q=   0.25000000 -0.43301270 -0.15353963
 q=   0.50000000  0.00000000  0.15353963
 q=  -0.50000000  0.00000000 -0.15353963
  reading force constants from file GaN.dyn12
  nqs=            6
 q=   0.25000000  0.43301270 -0.30707926
 q=   0.25000000 -0.43301270 -0.30707926
 q=  -0.25000000 -0.43301270 -0.30707926
 q=  -0.50000000  0.00000000 -0.30707926
 q=  -0.25000000  0.43301270 -0.30707926
 q=   0.50000000  0.00000000 -0.30707926

      q-space grid ok, #points =   64

      fft-check warning: sum of imaginary terms = 3.057932E-07
 
     Q2R          :      0.42s CPU      0.52s WALL

 
   This run was terminated on:  12:47:11  10Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
